### ISSUE TITLE
Do we really need this?

### ISSUE THREAD

**Issue by ghost** [quadrotor-IITKgp/git-sandbox#6]
Is this a repository that is necessary as part of the organization?
Can this be set up locally in the KGP network if required for fresher integration rather than cluttering the space while not being used, almost at all

**Comment by abhinavjain241** [comment]
Yeah I think this can be removed. It was created very early on for the purpose of teaching git.
